src/modules/dashboard/lists/SupplyAssetsList/WalletEmptyInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import { NetworkConfig } from 'src/ui-config/networksConfig';
+import { Warning } from 'src/components/primitives/Warning';
+import { Link } from '../../../../components/primitives/Link';
+import { Trans } from '@lingui/macro';
+import { ChainId } from '@aave/contract-helpers';
+
+type WalletEmptyInfoProps = Pick<NetworkConfig, 'bridge' | 'name'> & {
+  chainId: number;
+  icon?: boolean;
+};
+
+export function WalletEmptyInfo({ bridge, name, chainId, icon }: WalletEmptyInfoProps) {
+  const network = [ChainId.avalanche].includes(chainId) ? 'Ethereum & Bitcoin' : 'Ethereum';
+  return (
+    <Warning severity="info" icon={icon}>
+      {bridge ? (
+        <Trans>
```

Comment:
moved it in two seperate trans so the po file contains the full strings (otherwise can't be translated with correct grammar)